parser/parameters.py

```diff
@@ -49,5 +49,9 @@ def generate_exception(e: Exception, func_name: str) -> Exception:
 ANSI_RESTORE_CURSOR = "\0338"
 
 
+# Log File Constants
+MAX_FILE_CHARS          = 73333333
```

Comment:
What is the rationale behind choosing this number specifically?

Reply:
I added a comment. 

110,000,000 characters is around when the file crashes (with some leeway). However, when running the code with 100,000,00 set as the parameter we check the log file and it consistenly has 1.5 times the characters. Thus, we will stop logging at 100,000,000 / 1.5 characters so that the logfile will end up with around 110,000,000 characters and not crash.

Reply:
Wont this number (e.g., 110,000,000), be dependent on the computer its running on? Maybe it's better to split the log file even more granular?

Reply:
When I compared my laptop to the bay computer I noticed a similar range of values. It is true that a worse computer could actually have problems. let me change it to be half to account for ALL range of computers. People can change this paramter if they dont want so many files created if they run a long test.